sql/core/src/main/scala/org/apache/spark/sql/SQLConf.scala

```diff
@@ -36,6 +36,8 @@ private[spark] object SQLConf {
   val PARQUET_INT96_AS_TIMESTAMP = "spark.sql.parquet.int96AsTimestamp"
   val PARQUET_CACHE_METADATA = "spark.sql.parquet.cacheMetadata"
   val PARQUET_COMPRESSION = "spark.sql.parquet.compression.codec"
+  val PARQUET_BLOCK_SIZE = "spark.sql.parquet.blocksize"
+  val PARQUET_PAGE_SIZE = "spark.sql.parquet.pagesize"
```

Comment:
I think that parquet has conf keys for these, right? Users just pass those properties in the `Options` (`parameters` in `ParquetRelation2`). Then, we will need to change parquet relation to set those properties correctly.